lib/ioc-rc

```diff
@@ -543,10 +543,12 @@ __rc_jails () {
         local name="$(zfs get -H -o value org.freebsd.iocage:host_hostuuid \
                     $jail)"
         local boot="$(zfs get -H -o value org.freebsd.iocage:boot $jail)"
+        local _jail_hostid="$(zfs get -H -o value org.freebsd.iocage:hostid \
+                    $jail)"
         local priority="$(zfs get -H -o value org.freebsd.iocage:priority \
-                        $jail)"
```

Comment:
$jail)" was already on a separate line, i just lined it out with the two
above!


Op 15/09/15 om 17:53 schreef Brandon Schneider:
>
> In lib/ioc-rc
> <https://github.com/iocage/iocage/pull/226#discussion_r39527937>:
>
> >          local priority="$(zfs get -H -o value org.freebsd.iocage:priority \
> > -                        $jail)"
>
> Please add this line back and remove line 549.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/iocage/iocage/pull/226/files#r39527937>.
>